testrtc.py

```diff
@@ -27,13 +31,49 @@ def get(self):
     content = template.render({})
     self.response.out.write(content)
 
+class UploadHandler(blobstore_handlers.BlobstoreUploadHandler):
```

Comment:
Is it enough to add a "Access-Control-Allow-Origin" response header with "Referer" for HEAD and "Origin" for POST? Or should I add some server side checking?